lib/internal/loader/resolveRequestUrl.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+const {URL} = require('url');
+const internalCJSModule = require('internal/module');
+const NativeModule = require('native_module');
+const Path = require('path');
```

Comment:
Why not use destructuring to prevent monkey patching as has been done elsewhere? Either way `s/P/p/`.